lunch-application/src/router.js

```diff
@@ -14,6 +15,7 @@ const router = new VueRouter({
   routes: [
     { path: '/login', component: Login },
     { path: '/Home', component: Home },
```

Comment:
Make this path lowercase for consistency

Reply:
done